src/edie/evaluator.py

```diff
@@ -116,16 +120,29 @@ def metadata_evaluation_report_as_dataframe(self):
     def entry_report(self, dictionary_id):
         return self.report['dictionaries'][dictionary_id]['entry_report']
 
+    def visualize(self):
+        df = self.entry_evaluation_report_as_dataframe().drop('errors', axis=1)
+        df = df.apply(lambda x: x / x.max(), axis=0)
+        df['dict_type'] = 0
+        parallel_coordinates(df, "dict_type", axvlines=True)
+        plt.show()
+
     def _entry_report(self, dictionary_id: str, entry_report: dict, entry: Entry):
         retrieved_entry: JsonEntry = self._retrieve_entry(dictionary_id, entry)
         if retrieved_entry is not None:
             if retrieved_entry.errors:
                 self._add_errors(entry_report, retrieved_entry.errors)
             self._run_entry_metrics_evaluators(retrieved_entry)
+        else: #TODO: handle None case
```

Comment:
Is this a critical issue. In which case we should probably throw an exception. Otherwise we should issue a warning that the system has some incomplete behaviour

Reply:
Yes, this is the place where we handle the not supported formats, currently ontolex is missing i think, i will deal with this later on in a separate issue together with some other refactoring stuffs.